config/locales/de.yml

```diff
@@ -431,13 +434,15 @@ de:
     afternoon: Nachmittag
     evening: Abend
     morning: Morgen
+  thank_you: Vielen Dank🎉
   user_destroyed: 'Benutzer %{email} wurde erfolgreich gelöscht.'
   users: Benutzer
   voluntary_platform: Freiwilligen Plattform
   volunteer_created: Freiwillige wurde erfolgreich erstellt.
   volunteer_destroyed: 'Freiwillige wurde erfolgreich gelöscht.'
   volunteer_updated: Freiwillige wurde erfolgreich aktualisiert.
   volunteers: Freiwillige
+  volunteer_application: Freiwillige/n Bewerbung
```

Comment:
The reason why I stated this and the translation in the comment below explicitly was, the context. 
Ohterwhise it would have been "Freiwilligen Bewerbung erfassen".